app/src/main/java/org/thecommonsproject/android/commonhealth/sampleapp/fragments/ResourceListFragment.kt

```diff
@@ -86,22 +84,18 @@ class ResourceListFragment : Fragment() {
         recyclerView.layoutManager = LinearLayoutManager(requireContext())
 
         val progressBar = view.findViewById<ProgressBar>(R.id.progress_bar)
-        progressBar.visibility = View.VISIBLE
-        viewModel.viewModelScope.launch {
-            if (!viewModel.isCommonHealthAvailable(requireContext())) {
-                Toast.makeText(requireContext(), "Please make sure CommonHealth is installed and setup", Toast.LENGTH_LONG).show()
-                return@launch
+        progressBar.visibility = View.GONE
+//        viewModel.viewModelScope.launch {
```

Comment:
I'd probably just remove this entirely if we don't want it anymore?

Reply:
done